docs/documentation.md

```diff
@@ -0,0 +1,16 @@
+# Documentation 
+
+It follows a list of all the implemented workflows:
+
+- [`publish-next-to-ghpages`](../.github/workflows/publish-next-on-ghpages.yml):
```

Comment:
Make one file per workflow, and link them here:
```suggestion
- [`publish-next-to-ghpages`](./publish-next-to-ghpages.md):
```

Reply:
Also, be consistent in naming, you have both:
- `publish-next-to-ghpages` and
- publish-next-on-ghpages

Choose one and stick to it for all occurrences, otherwise you will potentially generate bugs for people copying from the wrong place